docs/sources/next/extensions/explore.md

```diff
@@ -215,10 +183,6 @@ Use the table to explore the many extensions. Questions? Feel free to join the d
         <h4>xk6-output-clickhouse</h4>
         <p>Export results to ClickHouse</p>
     </a>
-    <a href="https://github.com/dynatrace/xk6-output-dynatrace" target="_blank" class="nav-cards__item nav-cards__item--guide">
```

Comment:
The extension does not meet the registry requirements
https://github.com/Dynatrace/xk6-output-dynatrace/issues/15
https://github.com/Dynatrace/xk6-output-dynatrace/issues/14
https://github.com/Dynatrace/xk6-output-dynatrace/issues/13